symforce/codegen/codegen_config.py

```diff
@@ -20,6 +24,7 @@ class CodegenConfig:
         use_eigen_types: Use eigen_lcm types for vectors instead of lists
```

Comment:
Should `use_eigen_types` exist on the base class, as it does not exist in other languages. (I realize this is unchanged by your PR, but it seemed relevant to the change).

Reply:
It affects what we put in LCM definitions for generated LCM types regardless of target language (although of course the languages that currently support generated functions that take/return LCM types are currently just Python and C++ I think)